db.scheme

```diff
@@ -5,7 +5,8 @@ CREATE TABLE IF NOT EXISTS `person` (
 
 CREATE TABLE IF NOT EXISTS `restaurant` (
  `id`   INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
- `name` TEXT NOT NULL
+ `name` TEXT NOT NULL,
+ `comment` TEXT NOT NULL
```

Comment:
It is rather simple to apply the relevant change through sqliteBrowser.
We could use the whole "check for column and add if necessary" but that would be a very brittle update path if we ever decide to remove/change this column again. I say we postpone this until we have a DB schema version where we can define an upgrade path in a deterministic and simple way instead of inferring stuff from the available columns.